Content.Shared/Wieldable/SharedWieldableSystem.cs

```diff
@@ -166,6 +166,9 @@ private void OnUseInHand(EntityUid uid, WieldableComponent component, UseInHandE
             args.Handled = TryWield(uid, component, args.User);
         else if (component.UnwieldOnUse)
             args.Handled = TryUnwield(uid, component, args.User);
+
+        if (HasComp<UseDelayComponent>(uid) && !component.UseDelayOnWield)
+            args.ApplyDelay = false;
```

Comment:
Bit suss on why we're even mixing usedelay into wieldable and not using a 3rd system? Now forks are reliant upon keeping the 2 as they're coupled.

Reply:
idk they've always been coupled im just adding a toggle to bypass the use delay.
There's different uses where you want use delay to either activate on wield or activate on something else.